### PR TITLE
add support for thrift IDL

### PR DESCRIPTION
**Pull request by timfeirg**
it's mostly copied from proto.vim, but it works.

### CODE REVIEW

File: autoload/ctrlp/funky/ft/thrift.vim [hunk 1]
```diff
@@ -0,0 +1,13 @@
+" Language: Thrift
+" Author: timfeirg
+" License: The MIT License
+
+function! ctrlp#funky#ft#thrift#filters()
+  let filters = [
+        \ { 'pattern': '\m\C^[\t ]*struct[\t ]\+',
+        \   'formatter': [] },
+        \ { 'pattern': '\m\C^[\t ]*service[\t ]\+',
```
Comment (tacahiroy) [inline autoload/ctrlp/funky/ft/thrift.vim]: It is suggested gathering  these patterns into one for performance pov, coz Funky tries to find patterns one by one.
So in this case like this is probably faster:
```vim
'\m\C^[\t ]*\(struct\|service\)[\t ]\+'
```

**Comment by tacahiroy** [comment]
Thanks for the PR. I've commented on the code.

**Comment by timfeirg** [comment]
oh thanks, didn't realize that's a thing, it's now fixed

**Comment by tacahiroy** [comment]
Thanks for introducing a new file type.

**Pull request merged**